pkg/controller/kubelet-config/kubelet_config_nodes.go

```diff
@@ -242,7 +227,7 @@ func (ctrl *Controller) addNodeConfig(obj interface{}) {
 	if nodeConfig.Name != ctrlcommon.ClusterNodeInstanceName {
 		message := fmt.Sprintf("The node.config.openshift.io \"%v\" is invalid: metadata.name Invalid value: \"%v\" : must be \"%v\"", nodeConfig.Name, nodeConfig.Name, ctrlcommon.ClusterNodeInstanceName)
 		glog.V(2).Infof(message)
-		ctrl.updateNodeConfigDegradedStatus(nodeConfig, message, "UpdateProhibited")
+		ctrl.eventRecorder.Eventf(nodeConfig, corev1.EventTypeNormal, "ActionProhibited", message)
```

Comment:
For errors in the controller for nodes objects, other than events here, is there anywhere else this gets bubbled to?

Reply:
The ideal place for errors related to config node objects would have been the status filed of the node config object. But unfortunately, [we don't have it anymore](https://github.com/openshift/api/pull/1187).  We don't want to introduce any unnecessary coupling between `Machine Config Pool` and  `Worker Latency Profiles` by adding any `Worker Latency Profile` specific statuses there. 

So we thought of just emitting an event here so that we can passively let the user know that they are attempting a prohibited transition (e.g. `default` to `low` or vice-versa) and it has been rejected. IMO, this is slightly better than asking the user to look into the logs of the machine config controller.

Reply:
ack, thanks!